### DISCUSSION TITLE
Unable to completely remove LANraragi

### DISCUSSION THREAD

**Discussion by defski** [Difegue/LANraragi#668] [Q&A] [answered]
I've used the installer, and I've used Everything to find every remaining remnants of lanraragi, but my settings are still saved when I reinstall lanraragi. I've also removed every file but my doujins in my doujins folder. What on earth am I missing?

**Comment by Difegue** [comment, marked as answer]
The GUI app's settings are located in a separate folder since that uses WPF's built-in configuration support: You should be able to find them at `C:\Users\[Username]\AppData\Local\Karen`.
